autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml

```diff
@@ -4,8 +4,6 @@
     avoidance:
       resample_interval_for_planning: 0.3               # [m]
       resample_interval_for_output: 4.0                 # [m]
-      detection_area_right_expand_dist: 0.0             # [m]
```

Comment:
I checked this change is applied in the parameter file in the [file](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/config/avoidance/avoidance.param.yaml)
but is related to this [PR](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/config/avoidance/avoidance.param.yaml)?

Reply:
@kyoichi-sugahara Thanks for your checking.
I'm sorry fot the confusion. This diff is not related to https://github.com/autowarefoundation/autoware.universe/pull/4731 but is related to https://github.com/autowarefoundation/autoware.universe/pull/4525.

I forgot to remove unused params, and remove them in this PR togather.

Reply:
Ok i see :+1: 
Thanks!